Classes/KIFUITestActor.m

```diff
@@ -551,9 +557,10 @@ - (void)expectView:(UIView *)view toContainText:(NSString *)expectedResult
 - (void)clearTextFromFirstResponder
 {
     @autoreleasepool {
-        UIView *firstResponder = (id)[[[UIApplication sharedApplication] keyWindow] firstResponder];
-        if ([firstResponder isKindOfClass:[UIView class]]) {
-            [self clearTextFromElement:(UIAccessibilityElement *)firstResponder inView:firstResponder];
+        for (UIResponder *firstResponder in [[UIApplication sharedApplication] firstResponders]) {
+            if ([firstResponder isKindOfClass:[UIView class]]) {
+                [self clearTextFromElement:(UIAccessibilityElement *)firstResponder inView:(UIView *)firstResponder];
```

Comment:
We probably should only do this from one element and not multiple.

Reply:
Do you mean that I should revert this portion so it continues to only check `UIApplication.keyWindow`?

Reply:
I guess what I mean is to bail from the loop after clearing the text from some first responder. Probably would want to do the same as above and output a warning if there are multiple matching fields and we're only clearing the contents of one of them.